hack/arktos-up-scale-out-poc.sh

```diff
@@ -86,6 +92,27 @@ if [ "${ENABLE_POD_PRIORITY_PREEMPTION}" == true ]; then
 fi
 FEATURE_GATES="${FEATURE_GATES},WorkloadInfoDefaulting=true,QPSDoubleGCController=true,QPSDoubleRSController=true"
 
+# check for network service support flags
+if [ -z ${DISABLE_NETWORK_SERVICE_SUPPORT} ]; then # when enabled
+  # kubelet enforces per-network DNS ip in pod
+  FEATURE_GATES="${FEATURE_GATES},MandatoryArktosNetwork=true"
+  # tenant controller automatically creates a default network resource for new tenant
+  ARKTOS_NETWORK_TEMPLATE="${KUBE_ROOT}/hack/testdata/default-flat-network.tmpl"
+  #CNIPLUGIN is set to use flannel as default
+  CNIPLUGIN="flannel"
```

Comment:
is flannel the default cni for scale-out only when service support is enabled?

Reply:
yes.